src/provider/model/Toast.ts

```diff
@@ -265,12 +265,18 @@ export class Toast implements LayoutItem {
     public async setTransform(transform: ReadonlyRectangle): Promise<void> {
         Object.assign(this._origin, transform.origin);
 
-        return this.trackAnimation((await this._webWindow).setBounds({
+        // TODO [SERVICE-739] This is a fix for setBounds not working on hidden windows.
+        await (await this._webWindow).showAt(transform.origin.x, transform.origin.y);
```

Comment:
Can you take the `Promise.all` of this and the setBounds, and pass that promise into `trackAnimation`?

We need to make sure a promise is pushed into the queue (which happens inside `trackAnimation`) synchronously to avoid possible toast glitches.

Reply:
Using `Promise.all` now with the hide removed as once we reach this point the toast will be shown right after.